docs/user_guide/HOWTO/Puppetmaster_Backup.rst

```diff
@@ -11,8 +11,6 @@ This section details the steps required to back up the Puppet Master.
    the Puppet master. If a different method is preferred, the user must install
    and configure it first.
 
-.. WARNING::
-
 1. Backup ``/etc/puppetlabs/puppet/ssl``
 2. Backup ``/etc/puppetlabs/puppet``
 3. Backup ``/srv/rsync``
```

Comment:
Oof `/srv/rsync`

Reply:
@jeefberkey That wasn't part of this PR, but I can go ahead and fix it. This was left over by another commit.